packages/agoric-cli/src/chain-config.js

```diff
@@ -92,6 +97,8 @@ export function finishCosmosApp({
   rosettaPort = `${DEFAULT_ROSETTA_PORT}`,
 }) {
   const rpcPort = Number(portNum);
+  // TODO: Use an accurate narrow type.
+  /** @type {any} */
```

Comment:
We can at least narrow a tiny bit based on the TOML library we use:
```suggestion
  /** @type {Record<string, any>} */
```